### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by mulkieran**

### CODE REVIEW

**Comment by mulkieran** [comment]
rebased

**Comment by mulkieran** [comment]
rebased.

**Comment by mulkieran** [comment]
The changelog will get some more changes before it is done, but it seems like it has enough already, and I'ld like to get the statement about our cryptsetup requirememt laid down.

**Comment by jbaublitz** [comment]
We may be able to make the cryptsetup minimum version 2.2 depending on how #1888 shapes up.

**Comment by mulkieran** [comment]
> We may be able to make the cryptsetup minimum version 2.2 depending on how #1888 shapes up.

I'ld like to insist on 2.3 at least to some extent; we could make it recommended 2.3, requires 2.2.

**Comment by jbaublitz** [comment]
> > We may be able to make the cryptsetup minimum version 2.2 depending on how #1888 shapes up.
> 
> I'ld like to insist on 2.3 at least to some extent; we could make it recommended 2.3, requires 2.2.

That sounds like a good idea.

**Comment by mulkieran** [comment]
> > > We may be able to make the cryptsetup minimum version 2.2 depending on how #1888 shapes up.
> > 
> > 
> > I'ld like to insist on 2.3 at least to some extent; we could make it recommended 2.3, requires 2.2.
> 
> That sounds like a good idea.

We have to insist now, we can't compile w/out anymore.

**Comment by mulkieran** [comment]
rebased and updated.

**Comment by mulkieran** [comment]
rebased and updated.

**Comment by mulkieran** [comment]
This is up for more-or-less final review. Even if I have to add a few more entries, I won't put it up again, in order to manage the release more easilyt.

File: CHANGES.txt [hunk 1]
```diff
@@ -1,3 +1,217 @@
...
+Python import sorter: isort (4.3.4)
+
+New minimum dependency requirement:
+  - cryptsetup: 2.3
+  - libblkid: 2.32
+
+New minimum Rust crate requirement:
+  - dbus: 0.8
+  - libc: 0.2.68
+  - libcryptset-rs: 0.4.0
```
Comment (jbaublitz) [inline CHANGES.txt:19]: Looks like there's a typo.

File: CHANGES.txt [hunk 2]
```diff
@@ -1,3 +1,217 @@
...
+  https://github.com/stratis-storage/stratisd/pull/1936
+  https://github.com/stratis-storage/stratisd/pull/1933
+  https://github.com/stratis-storage/stratisd/pull/1931
+  https://github.com/stratis-storage/stratisd/pull/1930
+  https://github.com/stratis-storage/stratisd/pull/1924
+  https://github.com/stratis-storage/stratisd/pull/1922
+  https://github.com/stratis-storage/stratisd/pull/1910
+  https://github.com/stratis-storage/stratisd/pull/1841
+
+- Require libcryptseup bindings:
```
Comment (jbaublitz) [inline CHANGES.txt:81]: Typo

File: CHANGES.txt [hunk 3]
```diff
@@ -1,3 +1,217 @@
...
+  https://github.com/stratis-storage/stratisd/pull/1821
+
+- Add a new unstable Report interface for JSON formatted reports:
+  https://github.com/stratis-storage/stratisd/issues/1122
+  https://github.com/stratis-storage/stratisd/pull/2041
+  https://github.com/stratis-storage/stratisd/pull/2033
+  https://github.com/stratis-storage/stratisd/pull/2028
+  https://github.com/stratis-storage/stratisd/pull/1980
+
+- Require at least one blockdev for intialization actions:
```
Comment (jbaublitz) [inline CHANGES.txt:105]: Typo

**Pull request closed without merging**